airflow/dags/common_dag_tasks.py

```diff
@@ -47,7 +53,7 @@ def extract(dag_id,url) -> str:
 
 
 @task
-def transform(dag_id, models_subdir='staging',task_id="") -> None:
+def transform(dag_id, models_subdir='staging',task_id="",wait=False) -> None:
```

Comment:
Not sure why this wait parameter is added and then not used.

Reply:
I remember now - I at one point tried parametrizing the wait_for_completion aspect of the externaldagoperator, I can remove this.